src/coffee/product-type-import.coffee

```diff
@@ -29,10 +29,18 @@ class ProductTypeImporter
       @logger.bunyanLogger.debug = -> # noop
 
     @_ensureCredentials argv
-    .then (credentials) ->
+    .then (credentials) =>
       @sphereImporter = new ProductTypeImport.default(@logger, credentials)
       Promise.resolve @
 
+  _slugify: (text) =>
```

Comment:
you may try using this instead: https://epeli.github.io/underscore.string/. We use it in already in one of our importers: https://github.com/sphereio/sphere-node-product-csv-sync/blob/master/src/coffee/mapping.coffee#L62